test/linalg/symmetric.jl

```diff
@@ -296,10 +297,14 @@ end
         @testset "linalg binary ops" begin
             @testset "mat * vec" begin
                 @test Symmetric(asym)*x+y ≈ asym*x+y
+                xadj = x.'
+                @test xadj * Symmetric(asym).' ≈ xadj * asym
```

Comment:
Beyond the immediately following test, this test seems to check only that `transpose(Symmetric(asym))` is approximately `Symmetric(asym)`?